src/debugger/rnDebugSession.ts

```diff
@@ -91,48 +92,57 @@ export class RNDebugSession extends LoggingDebugSession {
+                            return this.rnCdpProxy.createServer()
+                                .then(() => {
+                                    logger.log(localize("StartingDebuggerAppWorker", "Starting debugger app worker."));
+
+                                    const sourcesStoragePath = path.join(this.projectRootPath, ".vscode", ".react");
+                                    // Create folder if not exist to avoid problems if
+                                    // RN project root is not a ${workspaceFolder}
+                                    mkdirp.sync(sourcesStoragePath);
+
+                                    // If launch is invoked first time, appWorker is undefined, so create it here
+                                    this.appWorker = new MultipleLifetimesAppWorker(
+                                        attachArgs,
+                                        sourcesStoragePath,
+                                        this.projectRootPath,
+                                        undefined);
+
+                                    this.appWorker.on("connected", (port: number) => {
+                                        logger.log(localize("DebuggerWorkerLoadedRuntimeOnPort", "Debugger worker loaded runtime on port {0}", port));
+
+                                        if (this.rnCdpProxy) {
+                                            const attachArguments = {
+                                                type: "pwa-node",
+                                                request: "attach",
+                                                name: "Attach",
+                                                port: port,
+                                                inspectUri: this.rnCdpProxy.getInspectUriTemplate(),
+                                            };
+
+                                            vscode.debug.startDebugging(
+                                                this.appLauncher.getWorkspaceFolder(),
+                                                attachArguments,
+                                                this.session
+                                            )
+                                            .then((childDebugSessionStarted: boolean) => {
+                                                if (childDebugSessionStarted) {
+                                                    resolve();
+                                                } else {
+                                                    reject(new Error("Can not start child debug session"));
+                                                }
+                                            },
+                                            err => {
+                                                reject(err);
+                                            });
+                                        } else {
+                                            throw new Error("CDP proxy doesn't work");
```

Comment:
Not sure if should show this abbreviation to user, also it doesn't look very informative.
Example : Cannot connect to debugger worker: Chrome debugger proxy is offline

Reply:
The error message was clarified. Now it sounds like: "Cannot connect to debugger worker: Chrome debugger proxy is offline".